lib/mjml/parser.rb

```diff
@@ -36,10 +36,14 @@ def render
     #
     # @return [String] The result as string
     # rubocop:disable Style/OptionalBooleanParameter: Fixing this offense would imply a change in the public API.
-    def run(in_tmp_file, beautify = true, minify = false, validation_level = 'strict')
+    # rubocop:disable Metrics/MethodLength:
+    # rubocop:disable Metrics/ParameterLists:
```

Comment:
I wasn't sure how to cope with these 2 rules. I guess fixing them would imply quite strong changes in code, and I'd rather not take that responsability